packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -448,9 +459,11 @@ export function reverseDataFrame(data: DataFrame): DataFrame {
     fields: data.fields.map((f) => {
       const values = [...f.values];
       values.reverse();
+
       return {
         ...f,
         values,
+        nanos: reverseNanos(f.nanos),
```

Comment:
this will add a `nanos: undefined` attribute to every field without nanos. if that is a problem, i can adjust the code to only assign the `nanos` attribute when it exists, but that will make the code longer. WDYT?